reconcile/openshift_base.py

```diff
@@ -443,6 +443,11 @@ def apply(
             # spec.selector changes
             current_resource = oc.get(namespace, resource_type, resource.name)
 
+            if current_resource["spec"]["strategy"]["type"] == "Recreate":
+                oc.delete(namespace, resource_type, resource.name)
+                oc.create(namespace, resource=annotated)
```

Comment:
@lechuk47 can this be done by `oc replace --force` to save one command process? Since delete and create/apply is used many places in code already, can be a future refactor.